.openpublishing.publish.config.json

```diff
@@ -12,7 +12,7 @@
       "locale": "en-us",
       "monikers": [],
       "moniker_ranges": ["common-js",
-                         "excel-js-preview", "excel-js-1.1", "excel-js-1.2", "excel-js-1.3", "excel-js-1.4", "excel-js-1.5", "excel-js-1.6", "excel-js-1.7", "excel-js-1.8", "excel-js-1.9",
+                         "excel-js-preview", "excel-js-1.1", "excel-js-1.2", "excel-js-1.3", "excel-js-1.4", "excel-js-1.5", "excel-js-1.6", "excel-js-1.7", "excel-js-1.8", "excel-js-1.9", "excel-js-1.10", "excel-js-online",
```

Comment:
If it's an easy change that won't have any downstream impact, consider changing the **preview** picklist item to **Preview** and the **online** item to **Online only**.   (And also update the **preview** picklist item for Outlook and Word as well.)

![image](https://user-images.githubusercontent.com/3019529/67342489-2bbdea00-f4e7-11e9-8c54-7c568e0d019f.png)